notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.ipynb

```diff
@@ -4,42 +4,80 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you add comments explaining the purpose of each block- for example, why certain keywords are extracted from the FITS header, and what each section of the code is intended to achieve.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Agreed, we have clarified the explanation for this cell and provided a link to Section 2.4 of the WFC3 Data Handbook, which provides a full list and explanation of the available header keywords. All other cells contain brief markdown explanations.